packages/utils/pick.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+module.exports = (object, props) => {
+  const result = {}
+  props.forEach(prop => {
+    if (prop in object) {
+      result[prop] = object[prop]
```

Comment:
this also takes from the prototype chain no ? unlike the "non-lodash pick" in the graphql file